borgmatic/config/schema.yaml

```diff
@@ -1087,6 +1087,21 @@ properties:
                         Password with which to connect to the restore database.
                         Defaults to the "password" option.
                     example: trustsome1
+                mysql_dump_command:
+                    type: string
+                    description: |
+                        Command to use instead of "mysqldump". This can be used
+                        to run a specific mysql_dump version (e.g., one inside 
+                        a running docker container). Defaults to "mysqldump".
+                    example: docker exec mysql_container mysqldump
+                mysql_command:
+                    type: string
+                    description: |
+                        Command to run instead of "mysql". This
+                        can be used to run a specific mysql
+                        version (e.g., one inside a running docker
+                        container). Defaults to "mysql".
```

Comment:
Would you mind wrapping this to 80 characters? Thanks.

Reply:
I'm not sure I follow. It appears to be under 80 characters per line already.

Reply:
It is less than 80, but it looks like these lines are wrapped to a _smaller_ character count, e.g. 70.

Reply:
Oh my bad! I thought it was good as long as it was under 80.

Reply:
No worries! It's not a big deal either way.